README.md

```diff
@@ -1,2 +1,95 @@
-# tests
-Contributed tests for Flutter
+# Flutter Tests
+
+This repository contains references to tests that are run with every
+commit to Flutter to verify that no breaking changes have been
+introduced.
```

Comment:
Specifically clarify that these tests are typically maintained by people outside of the Flutter team to give the Flutter team visibility into how their changes affect real-world developers using Flutter.